README.md

```diff
@@ -5,6 +5,16 @@
 This repository is for the OpenMRS Dispensing App. For more information, please see the
 [Design Documentation](https://wiki.openmrs.org/display/projects/Dispensing+Design+Components).
 
+## Required privileges
+
+Note that following privileges need to be installed and assigned to roles:
+
+* "Task: dispensing.create.dispense" - Allows user to Dispense Medication
+* "Task: dispensing.create.dispense.andModifyDetails" - Allows user to modify the Quantity, Drug, Formulation and Dose Instructions (from the values specified in the Order / Medication Request) when Dispensing
```

Comment:
I don't love this name.  Isn't this really "dispensing.allow.substitutions" or something like that?

Reply:
I do like that better, will tweak